example/lib/routes/forgot_password.dart

```diff
@@ -0,0 +1,135 @@
+                  minHeight: 4,
+                )
+              : SizedBox(
+                  height: 4,
+                ),
+          Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                Center(
+                  child: const Text(
+                    'Testing forgot password with login ID',
+                    style: TextStyle(fontSize: 15, fontWeight: FontWeight.bold),
+                  ),
+                ),
+                SizedBox(
+                  height: 16,
+                ),
+                Row(
+                  children: [
+                    const Text('Enter login id:'),
+                    SizedBox(
+                      width: 10,
+                    ),
+                    Expanded(
+                      child: TextField(
+                        controller: _loginIdEditingController,
+                      ),
+                    )
+                  ],
+                ),
+                SizedBox(
+                  height: 26,
+                ),
+                ButtonTheme(
+                  minWidth: 240,
+                  child: ElevatedButton(
+                    style: ElevatedButton.styleFrom(
+                      textStyle: TextStyle(
+                        color: Colors.white,
+                      ),
+                    ),
+                    onPressed: () {
+                      final String loginID =
+                          _loginIdEditingController.text.trim();
+                      sendRequest(loginID);
+                    },
+                    child: const Text('Send Request'),
+                  ),
+                ),
+                SizedBox(
+                  height: 20,
+                ),
+                Center(
+                  child: Text(
+                    _requestResult ?? '',
+                    style: TextStyle(
+                      fontWeight: FontWeight.bold,
+                      fontSize: 15,
+                    ),
+                  ),
+                )
+              ],
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  /// Submit request.
+  void sendRequest(loginId) async {
+    setState(() {
+      _inProgress = true;
+    });
+    GigyaSdk.instance.forgotPassword(loginId).then((result) {
+      debugPrint('Success');
+      setState(() {
+        _inProgress = false;
+        _requestResult = 'Requested: '
+            'instructions have been sent to your email';
+      });
+    }).catchError((error) {
+      setState(() {
+        _inProgress = false;
+        _requestResult = 'Request error\n\n${error.errorDetails}';
+      });
+    });
+  }
+}
+
+/// Specific response object used for the "accounts.isAvailableLoginID" endpoint.
+/// Extending the [GigyaResponse] object is optional if you require using its provided structure.
+class AvailableLoginIdResponse extends GigyaResponse {
```

Comment:
Remove if unused.
It is already registered on "send_request" route.

Reply:
Removed